packages/recommend-react/src/useRecommendations.ts

```diff
@@ -49,10 +51,12 @@ export function useRecommendations<TObject>({
       recommendClient,
       threshold,
       transformItems: transformItemsRef.current,
-    }).then((response) => {
-      setResult(response);
-      setStatus('idle');
-    });
+    })
+      .then((response) => {
+        setResult(response);
+        setStatus('idle');
+      })
+      .catch(throwAsyncError);
```

Comment:
pretty sure this is a breaking change, no?

Reply:
Indeed it is. This draft-PR is more as something we can link to in this error management discussion, rather than something I want to open right away.